docs/side_quests/metadata.md

```diff
@@ -0,0 +1,1039 @@
+                                      [ meta + [lang:lang], file ]
+                                  }
+                                  .map{ meta, file ->
+                                      def lang_group = (meta.lang.equals('de') || meta.lang.equals('en')) ? 'germanic' : 'romanic'
+                                      [ meta + [lang_group:lang_group], file ]
+                                  }
+                                  .view()
+
+    }
+
+    ```
+
+=== "Before"
+
+    ```groovy title="main.nf" linenums="20"
+    workflow  {
+
+      ch_samplesheet = Channel.fromPath("./data/samplesheet.csv")
+                      .splitCsv(header: true)
+                      .map { row ->
+                          [ [id:row.id, character:row.character], row.recording ]
+                      }
+
+      ch_prediction = IDENTIFY_LANGUAGE(ch_samplesheet)
+
+      ch_languages = ch_samplesheet.join(ch_prediction)
+                                  .map { meta, file, lang ->
+                                      [ meta + [lang:lang], file ]
+                                  }
+                                  .view()
+
+    }
+    ```
+
+Let's rerun it
+
+```bash title="View language groups"
+nextflow run main.nf -resume
+```
+
+```console title="View language groups"
+ N E X T F L O W   ~  version 24.10.4
+
+Launching `main.nf` [wise_almeida] DSL2 - revision: 46778c3cd0
+
+[da/652cc6] IDENTIFY_LANGUAGE (7) [100%] 7 of 7, cached: 7 ✔
+[[id:sampleA, character:squirrel, lang:fr, lang_group:romanic], /workspaces/training/side-quests/metadata/data/bonjour.txt]
+[[id:sampleB, character:tux, lang:de, lang_group:germanic], /workspaces/training/side-quests/metadata/data/guten_tag.txt]
+[[id:sampleC, character:sheep, lang:de, lang_group:germanic], /workspaces/training/side-quests/metadata/data/hallo.txt]
+[[id:sampleD, character:turkey, lang:en, lang_group:germanic], /workspaces/training/side-quests/metadata/data/hello.txt]
+[[id:sampleE, character:stegosaurus, lang:es, lang_group:romanic], /workspaces/training/side-quests/metadata/data/hola.txt]
+[[id:sampleF, character:moose, lang:fr, lang_group:romanic], /workspaces/training/side-quests/metadata/data/salut.txt]
+[[id:sampleG, character:turtle, lang:it, lang_group:romanic], /workspaces/training/side-quests/metadata/data/ciao.txt]
+```
+
+Let's understand how this transformation works. The `map` operator takes a closure that processes each element in the channel. Inside the closure, we're using a ternary operator to create a new language group classification.
+
+The ternary expression `(meta.lang.equals('de') || meta.lang.equals('en')) ? 'germanic' : 'romanic'` works like this:
+
+- First, it evaluates the condition before the `?`: checks if the language is either German ('de') or English ('en')
+- If the condition is true (language is German or English), it returns 'germanic'
+- If the condition is false (any other language), it returns 'romanic'
```

Comment:
I say just scrap this bit. You talk about ternaries above and point to docs, that's sufficient.